tests/unit/workflows/ruby_bundler/test_validator.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
+
+import mock
+from parameterized import parameterized
+
+from aws_lambda_builders.exceptions import MisMatchRuntimeError
+from aws_lambda_builders.workflows.ruby_bundler.validator import RubyRuntimeValidator
+
+class MockSubProcess(object):
+
+    def __init__(self, returncode, out=b"", err=b""):
+        self.returncode = returncode
+        self.out = out
+        self.err = err
+
+    def communicate(self):
+        return (self.out, self.err)
+
+class TestRubyRuntimeValidator(TestCase):
```

Comment:
can we add an integ test, or is the expectation that all existing integration tests will pass?